src/commonControls/listControl/ListControl.ts

```diff
@@ -1171,8 +1217,10 @@ export class ListControl extends Control implements InteractionModelContributor
                 this.evaluatePromptProp(act, this.props.reprompts.requestValue, input),
             );
 
-            const renderedAPL = this.evaluateAPLPropNewStyle(this.props.apl.requestValue, input);
-            this.addStandardAPL(input, builder, renderedAPL);
+            if (builder.aplMode !== APLMode.COMPONENT) {
```

Comment:
here and elsewhere, `=== APLMode.DIRECT` is more clear and future proof.

Reply:
This requires setting the APLMode.DIRECT on ControlManager, I was previously just setting aplMode to APLMode.COMPONENT only in new ComponentModeControlManager.
Updated.